lms/static/sass/views/_program-details.scss

```diff
@@ -451,13 +450,20 @@ $btn-color-primary: $primary-dark;
   }
 
   .upgrade-subscription {
-    margin: 10px 15px 10px 5px;
-    gap: 15px;
+    margin: 16px 0 10px;
+    row-gap: 16px;
+    column-gap: 24px;
+  }
+
+  .subscription-icon-launch {
+    width: 22.5px;
+    height: 22.5px;
+    margin-inline-start: 8px;
   }
 
   .subscription-icon-restart {
-    width: 24px;
-    height: 24px;
+    width: 22.5px;
+    height: 22.5px;
```

Comment:
I think we should communicate these values with the design team (Michelle) and follow the design guidelines here?

Reply:
sure